### PR TITLE
Add ambassador submission - David Biesack

### PR DESCRIPTION
**Pull request by DavidBiesack**
Add JSON Ambassador submission - David Biesack

### CODE REVIEW

File: programs/ambassadors/ambassadors.json [hunk 1]
```diff
@@ -1,24 +1,71 @@
...
-      "github": "iamthefirst",
-      "twitter": "iamthefirst",
-      "linkedin": "iamthefirst",
-      "company": "ACME",
+      "name": "David Biesack",
+      "img": "https://avatars.githubusercontent.com/u/545944?s=400&u=26818946106e2d8ae8c0cb5e0b72d6c7f612d268&v=4",
+        "bio": "Chief API Officer at APiture (I design banking APIs with OpenAPI 3.1, JSON Schema 2020/12) and author of the API Design Matters blog, https://apidesignmatters.substack.com/ ",
+      "title": "Chief API Officer",
+      "github": "DavidBiesack",
+      "twitter": "davdbiesack",
```
Comment (benjagm) [inline programs/ambassadors/ambassadors.json:8]:
```suggestion
      "mastodon": "@DavidBiesack@fosstodon.org",
```

**Review by benjagm — APPROVED** [review]
Thanks a lot for your application! We are so excited to have you onboard!

**Review by gregsdennis — APPROVED** [review]
Welcome David!

**Comment by DavidBiesack** [comment]
if this gets merged, how is the repo reset so other people can submit a PR ? The original template is gone.

Should I update the PR (and the `README.md` ) and leave `item[0]` in the array with the ambassador applicant/nomination template?

**Review by Julian — APPROVED** [review]
🎉

**Review by jviotti — APPROVED** [review]
Looks awesome. @DavidBiesack if you want, please feel free to add your articles to https://github.com/sourcemeta/awesome-jsonschema

**Review by Relequestual — APPROVED** [review]
Glad we can formally recognise people in our community! 🙌
Excited to have you onboard!

**Pull request merged**